### DISCUSSION TITLE
Nextjs v14: Unexpected token < in JSON at position 15396

### DISCUSSION THREAD

**Discussion by tnnquang** [vercel/next.js#57560] [Help]
### Summary

When I run the command dev or build or start, I see this error: SyntaxError: Unexpected token < in JSON at position 15396
    at JSON.parse (<anonymous>)
    at patchIncorrectLockfile (<**project path**>\node_modules\next\dist\lib\patch-incorrect-lockfile.js:92:33)

Link photo this error: https://ibb.co/vPLnXXN

If anyone has a solution to this problem, please help me remove this error message

**Comment by tkhv** [comment]
This can be caused by incorrect syntax in your package-lock.json. Try `npm install`, as that will attempt to patch your package-lock.json and may fix it.

**Reply by choidaruhan** [reply]
thanks bro!
